Applications/DataExplorer/VtkVis/VtkCompositeNodeSelectionFilter.h

```diff
@@ -28,7 +28,7 @@ class VtkCompositeNodeSelectionFilter : public VtkCompositeFilter
 
     void init() override;
 
-    /// Sets the point indeces to be highlighted
+    /// Sets the point indices to be highlighted
```

Comment:
This goes the whole way up to the interface and some other places => different PR.